Content.Shared/Atmos/Atmospherics.cs

```diff
@@ -236,6 +236,37 @@ static Atmospherics()
         /// </summary>
         public const float FrezonProductionConversionRate = 50f;
 
+        /// <summary>
+        ///     The minimum temperature N2O can decompose at.
+        /// </summary>
+        public const float N2ODecompositionMinTemp = 1400f;
```

Comment:
This is quite high. Could you explain why you picked this number?

Reply:
i didn't pick this number 🤷‍♂️ it was /tg devs who did